SDCAlertView/Source/SDCAlertControllerView.m

```diff
@@ -161,6 +161,7 @@ - (void)prepareForDisplay {
 	
 	[self addSubview:self.visualEffectView];
 	[self.visualEffectView sdc_alignEdgesWithSuperview:UIRectEdgeAll];
+	[self.scrollView invalidateIntrinsicContentSize];
```

Comment:
Before changes invalidateIntrinsicContentSize was called from layoutSubviews, but now it will not called automatically. Because of that we need to call it manually, for properly layout subviews.